MAINTENANCE.md

```
@@ -8,17 +8,26 @@ _Newest entries at top. Full git log for older entries._
 - Merged PR #75 (maintenance summary for 2026-06-30)
+- Fixed maintenance script to filter pip freeze output (avoid pulling in system packages like aptdaemon)
+- Closed duplicate PR #76 (superseded by #77)
 
-### PRs Opened in This Run
-- PR #76: Python dependency updates for showcase-servers/api-integration-hub
-- PR #77: Python dependency updates for showcase-servers/business-intelligence-mcp
-- PR #78: Python dependency updates for showcase-servers/content-automation-mcp
-- PR #79: Python dependency updates for showcase-servers/github-mcp-safe
-- PR #80: Python dependency updates for showcase-servers/social-distribution-mcp
-- PR #81: Python dependency updates for showcase-servers/social-poster-mcp
+### PR Status (opened this run)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)